index.html

```diff
@@ -179,6 +200,17 @@ <h3><dfn>Sink</dfn> interface</h3>
         events are used like described in the <code>Stream</code> interface. Depending on the use
         case, subtypes of <code>EventEmitter</code> or <code>Stream</code> are used.
       </p>
+      <p>
+        <dfn>SinkOptions</dfn> Sink constructor can take <code>options</code> object as parameter.
+      </p>
+      <div data-dfn-for="SinkOptions">
+        <p>
+          <dfn>baseIRI</dfn> Base IRI used to resolve or create relative IRIs.
+        </p>
+        <p>
+          <dfn>dataFactory</dfn> DataFactory that Sink will use if <code>import()</code> returns <code>Stream&lt;Quad&gt;</code>.
```

Comment:
Not sure I get this… why would Sink need that?

Reply:
This is needed if the stream is a text stream, for example if the sink is a parser.

Reply:
But then the explanation might be wrong; it's not about the output, but about the input then?

(Also confusing that we use Source and Sink also for text; we probably want something else.)

Reply:
Agreed, see #2.

Reply:
Few lines below we have

> Typical use cases
> Parser:
> Stream&lt;Quad&gt; import(Stream stream)

`import()` takes stream of text as argument and returns stream of quads. How would you explain that `dataFactory` will get used to construct those quads in the stream returned by `import()` ?

Reply:
`output` variable in this example: https://github.com/rdfjs/parser-jsonld#example

Reply:
Okay, works for me.

Reply:
Just the whole Stream/Stream<> is something we need to fix, but other issue.